daft/io/_csv.py

```diff
@@ -21,13 +21,14 @@
 def read_csv(
     path: Union[str, List[str]],
     schema_hints: Optional[Dict[str, DataType]] = None,
+    schema_override: Optional[Dict[str, DataType]] = None,
     has_headers: bool = True,
-    column_names: Optional[List[str]] = None,
     delimiter: Optional[str] = None,
     double_quote: bool = True,
     quote: Optional[str] = None,
     escape_char: Optional[str] = None,
     comment: Optional[str] = None,
+    flexible: bool = False,
```

Comment:
I think that the name should probably be more specified, e.g. `allow_variable_columns`, since `flexible` could apply to a lot of things, such as permissive decoding.